samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb

```diff
@@ -23,43 +23,47 @@ Module modMain
     Private Sub ShowGSpecifier()
         Console.WriteLine("G Specifier:")
         ' <Snippet1>
-        Console.WriteLine(ConsoleColor.Red.ToString("G"))           ' Displays Red
+        Console.WriteLine((CType(7, DayOfWeek)).ToString("G"))    ' Displays 7
+        Console.WriteLine(ConsoleColor.Red.ToString("G"))         ' Displays Red
         Dim attributes As FileAttributes = FileAttributes.Hidden Or _
                                            FileAttributes.Archive
-        Console.WriteLine(attributes.ToString("G"))     ' Displays Hidden, Archive                               
+        Console.WriteLine(attributes.ToString("G"))               ' Displays Hidden, Archive
         ' </Snippet1>
         Console.WriteLine()
     End Sub
 
     Private Sub ShowFSpecifier()
         Console.WriteLine("F Specifier:")
         ' <Snippet2>
-        Console.WriteLine(ConsoleColor.Blue.ToString("F"))         ' Displays Blue
+        Console.WriteLine((CType(7, DayOfWeek)).ToString("F"))    ' Displays 7
+        Console.WriteLine(ConsoleColor.Blue.ToString("F"))        ' Displays Monday, Saturday
```

Comment:
"Displays Monday, Saturday" doesn't fit here.

Reply:
Yes, you're correct. I've updated this...